examples/genomicsdb_query

```diff
@@ -335,7 +356,9 @@ def main():
                 df = gdb.query_variant_calls(query_protobuf=query_config, flatten_intervals=True)
                 df.to_csv(generate_output_filename(output, output_type, interval, idx), index=False)
             elif output_type == "json":
-                json_output = gdb.query_variant_calls(query_protobuf=query_config, json_output=json_output_mode.ALL)
+                json_output = gdb.query_variant_calls(
+                    query_protobuf=query_config, json_output=json_output_mode.SAMPLES_WITH_NUM_CALLS
```

Comment:
This is just an example so doesn't matter maybe...but I would think the `json_output_mode.ALL` is the more apt mode here since that returns all the variants.

I assume this was for the benchmarking -- maybe we add another output_type for that?